### DISCUSSION TITLE
Question about modeling a fiber tap

### DISCUSSION THREAD

**Discussion by JamesGaither** [netbox-community/netbox#10766] [Q&A]
Hi All,
I have a passive tap I am not quite sure of the best way to model in netbox. I was thinking to use the multi-terminated cables to allow a passthrough of a tap device, but it seems that a single rear port is only allowed to map to multiple front ports and not multi front to multi rear. I have attached a picture of the fiber tap mapping I am using. Any advice would be much appreciated. 

![image](https://user-images.githubusercontent.com/16848929/198218919-8ca894ba-4638-400c-b208-eb58aaed15b2.png)

**Comment by jeremystretch** [comment]
I'd model the A and B sides as front and rear ports to preserve end-to-end cable tracing, and model the tap port as an interface. Or maybe two interfaces, given that you don't typically receive data on both strands of a fiber pair.

**Reply by JamesGaither** [reply]
Thank you so much for the answer. I will give that a go today and see how it works out.
